git_perf/src/reporting.rs

```diff
@@ -182,6 +183,425 @@ impl PlotlyReporter {
         }
         None
     }
+
+    /// Add epoch boundary traces to the plot.
+    ///
+    /// These are vertical dashed gray lines where measurement epochs change.
+    /// Hidden by default (legendonly), user clicks legend to toggle visibility.
+    #[allow(dead_code)] // Will be integrated in Phase 2
+    pub fn add_epoch_boundary_traces(
```

Comment:
Only keep one version of the two methods for `add_epoch_boundary_traces` and adapt the tests.